CHANGELOG.md

```
@@ -5,15 +5,46 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/cryptomator/siv-mode/compare/1.6.0...HEAD)
+## [Unreleased](https://github.com/cryptomator/siv-mode/compare/1.6.1...HEAD)
+
+### Added
+- new lowlevel API:
+   * `new SivEngine(key).encrypt(plaintext, associatedData...)`
+   * `new SivEngine(key).decrypt(plaintext, associatedData...)`
+- implement JCA `Cipher` SPI:
+    ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)